### ISSUE TITLE
Switch to using accessible colours - a darker purple [Feature Request]

### ISSUE THREAD

**Issue by lukew-cogapp** [Laravel-Backpack/community-forum#355]
Hello,

We're using Backpack on a project but have noticed that the purple you use is not accessible on a white background (nor is the blue alternative).

They both fail to meet WCAG AA colour contrast ratio. Whilst we're aware that we can create overrides for these, it would be nice for the wider world (and especially those with any visual impairments) to have this available by default.

Alternatively, a simple-to-implement "primary colour" configuration variable would be great, so that we could change this to something accessible without needing to go through the _full_ custom config route.

Thank you.

Pasted image from https://webaim.org/resources/contrastchecker/:
![image](https://user-images.githubusercontent.com/83020016/210383307-47ba64b0-b6b9-4721-84df-30cccc0f4798.png)

**Comment by pxpm** [comment]
Hello @lukew-cogapp 

Thanks for the suggestion and the tip about the contrast, I've never thought about it, but after seeing it, you can't un-seen. It's like I want to change the main color that our theme uses NOW! 

Unfortunately can't do it, but for v6 we will sure pay more attention to those details in our color pallet. 
Also for v6 (due in a few months), we will be doing a major overhaul in Backpack themes sector, so that changing colors and stuff like that doesn't involve building steps. 

Out of curiosity I went to check about the purple in github Close with comment button:
![image](https://user-images.githubusercontent.com/7188159/210795194-a532bcd3-3824-4c0a-8573-550ad0eaa03a.png)

It seems they already know about https://webaim.org/resources/contrastchecker/ 🙃 
![image](https://user-images.githubusercontent.com/7188159/210795127-30e866b2-6c13-4e22-b5e8-c1bf7e0ddd69.png)

Thanks again for the suggestion @lukew-cogapp , it was a very good one I think 🙏

**Comment by lukew-cogapp** [comment]
Hi @pxpm 

Thanks for getting back to me! I absolutely understand that it's not possible to do now, but we eagerly await v6!

I'm happy to lend an opinion from an accessibility point of view at a later date. I can thoroughly recommend running backpack through this browser extension and you'll pick up the majority of issues without needing to read the full WCAG guidelines :)
